### ISSUE TITLE
Issue with rowspan table

### ISSUE THREAD

**Issue by Et3rnal** [simonbengtsson/jsPDF-AutoTable#427]
The following table will render fine in the browser:
```
<table class="table table-dark" id="my-table" >
  <tbody>
 <tr>
  <th rowspan="3">Details</th>
  <th>Name</th>
  <td>Sarmad</td>
 </tr>
 <tr>
  <th>ABN</th>
  <td>111000333</td>
 </tr>
 <tr>
  <th>Reg Add</th>
  <td>Tpwer 1</td>
 </tr>
  </tbody>
</table>
```

Using AutoTable {html: '#my-table'} to create the pdf, it seems to ignore one of the TDs

Currently, I solved the issue by adding an extra empty <td> tag on top
```
<table class="table table-dark" id="my-table" >
  <tbody>
 <tr>
  <th rowspan="3">Details</th>
  <th>Name</th>
  <td>Sarmad</td>
 </tr>
 <tr>
  <th></th>
  <th>ABN</th>
  <td>111000333</td>
 </tr>
 <tr>
  <th></th>
  <th>Reg Add</th>
  <td>Tpwer 1</td>
 </tr>
  </tbody>
</table>
```
However that will break the HTML table.

**Comment by keesbaake** [comment]
You're mixing th's and td's. What happens when you put all th's in a thead tag?

**Comment by Et3rnal** [comment]
The same

**Comment by RileyDavidson-Evans** [comment]
@Et3rnal I'm having a similar issue, have you found any solutions for this one ?

**Comment by RileyDavidson-Evans** [comment]
what I get when I print

<img width="703" alt="screen shot 2019-01-28 at 12 49 44 pm" src="https://user-images.githubusercontent.com/15188161/51863564-5fc8fd80-230f-11e9-89f5-6459101a1859.png">

**Comment by RileyDavidson-Evans** [comment]
what it should look like ( as visible in the browser )

<img width="976" alt="screen shot 2019-01-28 at 12 49 34 pm" src="https://user-images.githubusercontent.com/15188161/51863595-71aaa080-230f-11e9-8e15-52e32353ea7c.png">

**Comment by RileyDavidson-Evans** [comment]
Nested Tables also do not seem to work ( which makes sense )

**Comment by RileyDavidson-Evans** [comment]
working on this right now, will put PR up when done

**Comment by RileyDavidson-Evans** [comment]
#441

**Comment by RileyDavidson-Evans** [comment]
@Et3rnal  once the PR is merged please pull latest and try again !

**Comment by RileyDavidson-Evans** [comment]
woooooooooooops my bad 

new PR #443

**Comment by simonbengtsson** [comment]
Fixed #445 now which is most likely the same issue!
